addons/point_of_sale/static/tests/tours/product_configurator_tour.js

```diff
@@ -75,3 +75,20 @@ registry.category("web_tour.tours").add("ProductConfiguratorTour", {
             Chrome.endTour(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosProductWithDynamicAttributes", {
+    checkDelay: 50,
```

Comment:
Avoid `checkDelay` if possible. It normally hides missing steps when testing locally which results to random runbot error.